config.json

```diff
@@ -1316,24 +1316,24 @@
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-perf",
-        "version": "20.3.0",
-        "nugetVersion": "120.3.0",
+        "version": "20.2.0",
+        "nugetVersion": "120.2.0",
         "nugetId": "Xamarin.Firebase.Perf",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-perf-ktx",
-        "version": "20.3.0",
-        "nugetVersion": "120.3.0",
+        "version": "20.2.0",
```

Comment:
Version downgrade

Reply:
I specifically asked to review and merge this PR as 1st (or after last GPS one)

From discord:

> is there any way to commit #730 and #731 first?  the issue is that #729 also contains the changes in the other 2